### PR TITLE
feat: Implement Withdrawal event

### LINKED ISSUE #233: Standardize event data for withdraw event
1.  Read our [Contributor Guidelines](https://github.com/CarmineOptions/derisk-research/blob/master/CONTRIBUTING.md) and [README](https://github.com/CarmineOptions/derisk-research/blob/master/README.md).
2. Parse data from event which fetched on-chain to humman-readable format, for example, now we have `user = data[0]` and we want to have `user = data.user`
 - In method [parse_withdrawal_event](https://github.com/CarmineOptions/derisk-research/blob/master/data_handler/handler_tools/data_parser/zklend.py#L23C9-L23C31)
 - Event data structure - [event data](https://starkscan.co/event/0x03472cf7511687a55bc7247f8765c4bbd2c18b70e09b2a10a77c61f567bfd2cb_4)
3. Create Serializers model using pydantic in [serializers.py](https://github.com/CarmineOptions/derisk-research/blob/master/data_handler/handler_tools/data_parser/serializers.py)
4. In method [process_withdrawal_event](https://github.com/CarmineOptions/derisk-research/blob/master/data_handler/handlers/loan_states/zklend/events.py#L254)
 substitute fetching data by index `Example: user= data[0]` to `data.user` using serializer class

### PR DESCRIPTION
**Pull request by Mystic-Nayy**
Resolves #233

### CODE REVIEW

File: get-pip.py
Comment (djeck1432) [inline get-pip.py:1]: Why do we have this file here? 
Please remove it

**Review by djeck1432 — CHANGES_REQUESTED** [review]

**Review by djeck1432 — CHANGES_REQUESTED** [review]
Update also in `events.py` logic 
please, read issue description

**Comment by djeck1432** [comment]
@Mystic-Nayy Update also in events.py logic
please, read issue description
@Mystic-Nayy Please resolve conflict

**Comment by Mystic-Nayy** [comment]
> @Mystic-Nayy Update also in events.py logic please, read issue description @Mystic-Nayy Please resolve conflict

I am on it sir

**Comment by djeck1432** [comment]
@Mystic-Nayy Update also in events.py logic
please, read issue description
@Mystic-Nayy Please resolve conflict

**Review by djeck1432 — CHANGES_REQUESTED** [review]
Looks good to me, resolve please conflict

**Comment by djeck1432** [comment]
@Mystic-Nayy Please, resolve conflict

File: data_handler/handler_tools/data_parser/zklend.py [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+        Args:
+            event_data (list[Any]): A list containing the raw event data, typically with 3 or more elements:
+                user address, amount withdrawn, token address, and additional data.
+
+        Returns:
+            WithdrawalEventData: A Pydantic model with the parsed and validated event data in a human-readable format.
+        """
+        
+        parsed_event = WithdrawalEventData(
```
Comment (djeck1432) [inline data_handler/handler_tools/data_parser/zklend.py:71]: remove this `parsed_event` and return directly `WithdrawalEventData`

File: data_handler/handler_tools/data_parser/serializers.py [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+
+        Returns:
+            Decimal: Converted decimal value.
+        """
+        try:
+            return Decimal(int(value, base=16))
+        except ValueError:
+            raise ValueError("%s field is not a valid hexadecimal number" % info.field_name)
+
+class WithdrawalEventData(BaseModel):
```
Comment (djeck1432) [inline data_handler/handler_tools/data_parser/serializers.py:91]: add docstring

**Review by djeck1432 — CHANGES_REQUESTED** [review]
small issues

**Comment by djeck1432** [comment]
@Mystic-Nayy I resolved merge conflict, fix small issues

**Comment by djeck1432** [comment]
@Mystic-Nayy and please, tell me how did you test it?

**Review by djeck1432 — APPROVED** [review]
Looks good to me

**Pull request merged**